api/actions/views.py

```diff
@@ -98,34 +110,21 @@ def complete(request, action_id):
     volunteer = request.user.volunteer
     action = get_object_or_404(Action, pk=action_id)
 
-    if action.action_status != ActionStatus.ASSIGNED or action.volunteer != volunteer:
-        return redirect('actions:detail', action_id=action.id)
+    # if action.action_status != ActionStatus.ASSIGNED or action.volunteer != volunteer:
+    #     return redirect('actions:detail', action_id=action.id)
```

Comment:
Do we still need this?

Reply:
Oh dear yeah, forgot to re-activate it after I removed it to be able to re-submit the form multiple times, sorry :) This prevents the user from seeing the form if the action is already completed or if it's not the right volunteer.